.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
```

Comment:
Celowo podpinasz travisa? Ja bym polecił ci github actions. Jak już podpinasz to wyklikaj po stronie travisa hooki żeby się pokazywały na githubie buildy. Zerknij jak to wygląda w innych projektach przy użyciu github actions: https://github.com/TouK/nussknacker/pull/1445/checks